ledger-tool/src/main.rs

```diff
@@ -2505,17 +2509,21 @@ fn main() {
                             },
                         );
 
-                        let mut unchanged_accounts = stake_calcuration_details
-                            .keys()
-                            .collect::<HashSet<_>>()
+                        // Ensure DashMap lock is dropped
```

Comment:
nits: `// Ensure DashMap lock is dropped by copying the actual inner pubkeys out of it`